docs/developer_guide.md

```diff
@@ -45,6 +45,15 @@ website and mixer changes.
   ./run_test.sh --setup_python
   ```
 
+- Python 3.12
+
+  You need to run the following command:
+  ```brew install python-setuptools
```

Comment:
should be:

(Replace ' with ` below, otherwise the comment itself renders the markdown)

'''bash
brew install python-setuptools
'''

Reply:
done.